src/winter_onboarding_2021/fleet_management_service/db/cab.clj

```diff
@@ -0,0 +1,28 @@
+(ns winter-onboarding-2021.fleet-management-service.db.cab
+  (:require [next.jdbc.sql :as sql]
+            [winter-onboarding-2021.fleet-management-service.db.core :as db]
+            [next.jdbc.result-set :as rs]
+            [camel-snake-kebab.core :as csk]
+            [honeysql.core :as honeysql]))
+
+(def sql-opts {:builder-fn rs/as-kebab-maps
+               :column-fn  csk/->snake_case_string
+               :table-fn   csk/->snake_case_string})
+
+(defn create [cab]
+  (sql/insert! db/db-conn :cabs cab sql-opts))
+
+(defn sqlmap [offset limit]
+  {:select [:*]
+   :from   [:cabs]
+   :limit (keyword (str limit))
+   :offset (keyword (str offset))})
+
+(defn select
+  ([] (sql/query db/db-conn
+                 (honeysql/format (sqlmap 0 10))
+                 sql-opts))
+  ([offset limit] (sql/query db/db-conn
```

Comment:
On UI side, the user will be shown `page size` and `page number`.
Won't it be better if `select` just received `page-size` & `page-number` and then `select` function can calculate the `offset` according to that?
Will save calculation on the client side.